docs/additional-resources/openms-git-workflow.md

```diff
@@ -0,0 +1,197 @@
+OpenMS Git Workflow
+===================
+
+Before getting started, install latest version of git to avoid problems like GitHub https authentication errors
+(see [Troubleshooting cloning errors](https://docs.github.com/en/repositories/creating-and-managing-repositories/troubleshooting-cloning-errors) and a solution using [ssh](https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories)).
+
+OpenMS follows the [git flow workflow](https://nvie.com/posts/a-successful-git-branching-model/). The difference is that
+merge commits are managed via pull requests instead of creating merge commits locally.
+
+## Naming conventions
+
+Naming conventions for the following apply:
+
+* A **local repository** is the repository that lies on your hard drive after cloning.
+* A **remote repository** is a repository on a git server such as GitHub.
+* A **fork** is a copy of a repository. Forking a repository allows you to freely experiment with changes without affecting the original project.
+* **Origin** refers to a remote repository that you have forked. Call this repository `https://github.com/_YOURUSERNAME_/OpenMS`.
+* **Upstream** refers to the original remote OpenMS repository. Call this repository `https://github.com/OpenMS/OpenMS`.
+
+## Create fork
+
+Start by [forking](https://docs.github.com/en/get-started/quickstart/fork-a-repo) the OpenMS repository.
+
+To create a fork, click **Fork** under the main menu as shown below.
+
+![image info](../images/additional-resources/click-fork.png)
+
+## Clone your fork
+
+To obtain a local repository copy, clone your fork using:
+
+```bash
+$ git clone https://github.com/_YOURUSERNAME_/OpenMS.git
```

Comment:
This is also kind of a nitpick but I think it's better to use the SSH method for cloning so it would be something like
```bash
git clone git@github.com:_YOURUSERNAME_/OpenMS.git
```
Last time I cloned a repo with https, git wouldn't let me push to remote. I think since you gave the link to creating a token earlier, this is probably the better method to share as the users are less likely to run into problems.

Reply:
very good point. The git is kind of outdated. In principle both ssh and https should work.

Reply:
for anything ssh, you might want to make sure SSH keys work and are synced properly; for GitHub, adding to your GitHub account. 

https://docs.github.com/en/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account

Reply:
(added a note)